### PR TITLE
Work around bug in qt package for Python 3.9

### PR DESCRIPTION
**Pull request by matthiasdiener**
Required for https://github.com/illinois-ceesd/mirgecom/pull/525

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -186,6 +186,9 @@ fi
 # Required for Nvidia GPU support on Linux (package does not exist on macOS)
 [[ $(uname) == "Linux" ]] && conda install --yes pocl-cuda
 
+# Necessary to build mpi4py
+[[ $(uname) == "Linux" ]] && conda install --yes gcc_linux-64
```
Comment (inducer) [inline install.sh:190]: What happens if you don't have this available? What about macOS? What about Linux+Power9?

Comment (matthiasdiener) [reply install.sh]: I found a better workaround that "fixes" the qt package that is causing this issue. https://github.com/conda-forge/qt-feedstock/issues/208 has more context.

**Comment by matthiasdiener** [comment]
Confirmed to work with https://github.com/illinois-ceesd/mirgecom/pull/525. This is ready for another review @inducer.

**Review by inducer — APPROVED** [review]
Thanks for investigating and finding the actual root cause! LGTM!

**Pull request merged**